src/componentes/journal/editor.js

```diff
@@ -26,7 +26,7 @@ export { convertJEditorData2Text };
  
 
 //2020-01-18
-export const JEditor = ({ ymd, range, onClose, saveTrigger, hintTriggerRef, onLoaded, redirect }) => {
+export const JEditor = ({ ymd, range, onClose, saveTrigger, hintTriggerRef, saveTriggerRef, onLoaded, redirect }) => {
```

Comment:
here too, the saveTrigger holds the same reference as saveTriggerRef, you can delete that saveTriggerRef since it is already being passed by the saveTrigger parameter.